README.md

```diff
@@ -24,12 +24,18 @@ npm install mime-db
 ### Database Download
 
 If you're crazy enough to use this in the browser, you can just grab the
-JSON file using [RawGit](https://rawgit.com/). It is recommended to replace
-`master` with [a release tag](https://github.com/jshttp/mime-db/tags) as the
-JSON format may change in the future.
+JSON file using [unpkg](https://unpkg.com/mime-db/db.json).
```

Comment:
Sorry I didn't see this one the first time. I think this should just be a link to the unpkg page for folks to learn more, rather than clicking a link to the json, right?